src/types/point.js

```diff
@@ -1,6 +1,8 @@
 import {Element} from 'chart.js';
 import {drawPoint} from 'chart.js/helpers';
-import {inPointRange, getElementCenterPoint, resolvePointPosition, setBorderStyle} from '../helpers';
+import {inPointRange, getElementCenterPoint, resolvePointPosition, setBorderStyle, setShadowStyle, removeShadowStyle, isImageOrCanvas} from '../helpers';
+
+const isFillablePointStyle = (ps) => !(typeof ps === 'string') || ps === 'circle' || ps === 'triangle' || ps.startsWith('rect');
```

Comment:
You probably guessed, I don't like this.

What about separate fill / border (or line) shadow settings, so the responsibility is transferred to the user.
Eg. `fillShadowColor` only to fill and `borderShadowColor` to stroke?

Reply:
Yes, I thought you don't like it.
Yesterday during developing, I was thinking about a `type` but I think fill/border shadow color could be easier to understand.
Let me change it and we will see how it will look like!

Reply:
maybe `fillShadowColor` and `strokeShadowColor` ? Sounds to me closer to the context operations.
And  what about `backgroundShadowColor` and `borderShadowColor` ?

Reply:
I think we are using background/border to be closer to css. Probably good to stick with those, but I'm not very confident about this.

Reply:
used background/border shadow color